### PR TITLE
K8up Project Governance

### PR DESCRIPTION
**Pull request by tobru**
## Summary

Describe the K8up project governance, as required by CNCF Sandbox onboarding:

* https://github.com/cncf/toc/issues/753

## Checklist

- [X] Keep pull requests small so they can be easily reviewed.
- [X] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`,
      as they show up in the changelog
- [X] Link this PR to related issues.

### CODE REVIEW

File: docs/modules/ROOT/pages/about/contribution_guide.adoc [hunk 1]
```diff
@@ -7,3 +7,68 @@ help this project come to fruition.
...
+
+Use your real name (sorry, no pseudonyms or anonymous contributions).
+
+If you set your `user.name` and `user.email` git configs, you can sign your commit automatically with `git commit -s`.
+
+Some K8up contributors might not be familiar with `git`, or have used a web based editor, and thus asking them to `git commit --amend -s` is not the best way forward.
+
+In this case, maintainers can update the commits based on clause (c) of the DCO.
+The most trivial way for a contributor to allow the maintainer to do this, is to add a DCO signature in a pull requests's comment, or a maintainer can simply note that the change is sufficiently trivial that it does not substantially
+change the existing contribution - i.e., a spelling change.
```
Comment (ccremer) [inline docs/modules/ROOT/pages/about/contribution_guide.adoc:71]:
```suggestion
The most trivial way for a contributor to allow the maintainer to do this, is to add a DCO signature in a pull requests's comment, or a maintainer can simply note that the change is sufficiently trivial that it does not substantially change the existing contribution - i.e., a spelling change.
```

**Review by ccremer — APPROVED** [review]

**Comment by Kidswiss** [comment]
Do we implement a DCO check again on pull requests?

**Comment by tobru** [comment]
> Do we implement a DCO check again on pull requests?

Yes, it is part of the Sandbox onboarding process.

**Review by Kidswiss — APPROVED** [review]
LGTM

**Pull request merged**